### DISCUSSION TITLE
Any idea to add the amazing new content-visibility prop?

### DISCUSSION THREAD

**Discussion by frederikhors** [tailwindlabs/tailwindcss#2162] [Ideas]
Any idea to add the amazing new https://web.dev/content-visibility/ prop to Tailwind?

**Comment by Cherry** [comment]
If the Tailwind team is interested, I'd be willing to spend some time implementing this into core. I've already created a plugin for my own use at https://github.com/Cherry/tailwindcss-content-visibility.
